security/s2a/internal/crypter/expander_test.go

```diff
@@ -0,0 +1,66 @@
+package crypter
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"google.golang.org/grpc/security/s2a/internal/crypter/testutil"
+	"testing"
+)
+
+type hkdfTestVector struct {
+	comment        string
+	prk, info, okm []byte
+	length         int
+}
+
+func TestExpanderExpansionInvalidLength(t *testing.T) {
```

Comment:
This test is not needed to be separate. Its test case can be merged with `TestExpand`. You might want to add a boolean `success` to the test vector to know if the test case is support to fail or succeed.

Reply:
This test is no longer valid since `length` is no longer a parameter so I removed it altogether.